src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -39,6 +39,9 @@ type Props = {
 
     /** Optionally apply "aria-hidden": "true" to CollapsibleSidebar wrapper */
     isHidden?: boolean,
+
+    /** Whether to show the collapsed version with only icons */
```

Comment:
I'm not familiar with how this is different than `expanded=false`. are `expanded` and `narrowItems` mutually exclusive?

Reply:
In `CollapsibleSidebar.scss` we have a media query that sets the sidebar width to the narrow width (`68px`) when the viewport width is under `1220px`. But above `1220px` it shrinks the sidebar with `expanded=false`, but it leaves the width causing the Files List to be displaced:

![app hvenkatesh inside-box net_folder_8803997002_s=5j4mo9gvb7mfpvyfjfvj5dxla5ivtjli (3)](https://user-images.githubusercontent.com/6400298/227012566-286f6e79-17a9-488f-81c9-b7e0fbf63328.png)

https://github.com/box/box-ui-elements/blob/90e7c476bd7f74c3ad15a49c069f5c5e823f8308/src/features/collapsible-sidebar/CollapsibleSidebar.scss#L115-L137

Question...am I re-creating how `expanded=false` should be working but isn't? ie show the collapsed narrow version regardless of screen size? What is the expected behavior of the `expanded` prop?